app/models/employee.rb

```diff
@@ -3,6 +3,7 @@ class Employee < ApplicationRecord
   has_many :received_kudos, class_name: 'Kudo', foreign_key: 'receiver', inverse_of: :receiver, dependent: :destroy
   has_many :orders, dependent: :destroy
   has_many :rewards, through: :orders
+  belongs_to :address, optional: true
```

Comment:
I would make the association another way - what if later user can have multiple addresses?

Reply:
Thanks :) I have changed direction of the association 👍